src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to a stylesheet with a `.scss` extension won't work properly in a browser. You should compile SCSS to CSS and link to the compiled `.css` file instead.